jax/_src/api.py

```diff
@@ -2480,7 +2480,11 @@ def make_jaxpr_f(*args, **kwargs):
       return closed_jaxpr, tree_unflatten(out_tree(), out_shapes_flat)
     return closed_jaxpr
 
-  make_jaxpr_f.__name__ = f"make_jaxpr({make_jaxpr.__name__})"
+  make_jaxpr_f.__module__ = "jax"
+  if hasattr(fun, "__qualname__"):
+    make_jaxpr_f.__qualname__ = f"make_jaxpr({fun.__qualname__})"
+  if hasattr(fun, "__name__"):
```

Comment:
should this be elif?

Reply:
No, objects may define both `__name__` and `__qualname`, or only one, or neither.